src/app/treasure-hunt/operation-costs/operation-costs.component.ts

```diff
@@ -313,7 +313,7 @@ export class OperationCostsComponent implements OnInit {
         energyType: 'electricity',
         energySource: '',
         fuelType: '',
-        totalEmissionOutputRate: this.convertUnitsService.value(this.globalSettings.totalEmissionOutputRate).from('MWh').to('kWh'),
+        totalEmissionOutputRate: this.globalSettings.totalEmissionOutputRate/1000,
```

Comment:
Set a new variable above ln 312 with a name describing the conversion or at least 'let converted___ = " and set the object.totalEmissionOutputRate from that.